### PR TITLE
feature: Make API version configurable

### PR DESCRIPTION
**Pull request by FezVrasta**
I'm sorry, I never written a single line of Python nor I know how to write HA components...

I think the code changes are fine, I'm not sure if I need to do something more to read the `api_version` configuration option from the `configuration.yaml`

This should be good enough to add support for v5 API and start experimenting with it and see what else needs to be fixed.

The usage would be:

```yaml
media_player:
  - platform: philips_js
    host: 10.0.0.2
    api_version: 5
```    

I guess once (if) this gets merged I'll need to update the HA component as well to make it accept this new parameter.

### CODE REVIEW

File: haphilipsjs/__init__.py [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
 
 LOG = logging.getLogger(__name__)
-BASE_URL = 'http://{0}:1925/1/{1}'
+BASE_URL = 'http://{0}:1925/{1}/{2}'
 TIMEOUT = 5.0
 CONNFAILCOUNT = 5
 
 class PhilipsTV(object):
-    def __init__(self, host):
+    def __init__(self, host, api_version="1"):
```
Comment (FezVrasta) [inline haphilipsjs/__init__.py]: Is this the correct way to define a default argument value?

**Comment by danielperna84** [comment]
Depending on what should be done the version could be passed along this way. But this change would have no effect at all since it doesn't change anything about the behavior of the module.

_Edit:_
Oh no, wait it's acutally doing something. I'll take a deeper look.

**Comment by danielperna84** [comment]
Do it like this:

At the top below `CONNFAILCOUNT` add:
`DEFAULT_API_VERSION = 1`

Then change `def __init__(self, host, api_version="1"):` to `def __init__(self, host, api_version=DEFAULT_API_VERSION):`.

The `BASE_URL` (and the rest you have changed) can stay as it is.

**Comment by FezVrasta** [comment]
Done!

**Comment by FezVrasta** [comment]
Thanks! May you release the new version so I can proceed with the other needed work on HA?

**Comment by danielperna84** [comment]
Done.

You need to change [this](https://github.com/home-assistant/home-assistant/blob/dc8424032bf813fb597ab62baab852ce6535dda8/requirements_all.txt#L335) and [this](https://github.com/home-assistant/home-assistant/blob/dc8424032bf813fb597ab62baab852ce6535dda8/homeassistant/components/media_player/philips_js.py#L22) to `0.0.2`.

[Here](https://github.com/home-assistant/home-assistant/blob/dc8424032bf813fb597ab62baab852ce6535dda8/homeassistant/components/media_player/philips_js.py#L42) you have to add the optional option for the API version. Like in the module you should also set a global `DEFAULT_API_VERSION` which will be used as the default fallback value. I think there's a `cv.number` validator available. Have a look at other platforms that use numbers.

The rest should be straightforward and depends on what you're doing.

**Pull request merged**